src/main/java/gov/usgs/wma/waterdata/collections/observations/ObservationTypeJSON.java

```diff
@@ -0,0 +1,20 @@
+package gov.usgs.wma.waterdata.collections.observations;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@JsonPropertyOrder(value={"name","url"})
+@Schema(description="Observation dataset in JSON format.")
+public class ObservationTypeJSON {
+	@Schema(description="observation dataset name", example="statistical-time-series")
+	public String getName() {
+		return null;
+	}
+
+	@Schema(description="Href link to the dataset.")
+	public String getUrl() {
+		return null;
+	}
```

Comment:
double checking that you want to return null here (and above)

Reply:
yes, that is the pattern for the swagger schema classes. This class is only used by
swagger.